content/spin/v2/rust-components.md

```diff
@@ -328,7 +330,7 @@ mod api {
 }
 ```
 
-> For further reference, see the [Spin SDK HTTP router](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/http/struct.Router.html).
+> The Spin SDK HTTP router has additional async methods. For example, the `get_async` and `post_async` methods register an async handler at the path for the HTTP GET and HTTP POST methods. For further reference, see the [Spin SDK HTTP router](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/http/struct.Router.html).
```

Comment:
I would prefer not to make the "for further reference" seem specific to the async case.  What if we broke this into two paragraphs, e.g.

```suggestion
Handlers within a `Router` can be sync or async. Use `Router`'s "plain" methods (e.g. `get`, `post`) to assign synchronous handlers, and its "async" methods (e.g. `get_async`, `post_async`) for asynchronous handlers.  You can mix sync and async handlers in the same `Router`, and can use `handle` or `handle_async` to invoke `Router` processing, regardless of whether invididual handlers are sync or async.

> For further reference, see the [Spin SDK HTTP router](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/http/struct.Router.html).
```

I also added some detail about the relationship between sync and async stuff that the original questioner was seeking clarification about.